node/src/chain_spec.rs

```diff
@@ -53,10 +53,56 @@ fn get_bootnodes() -> Vec<MultiaddrWithPeerId> {
     ]
 }
 
-pub fn development_config() -> Result<ChainSpec, String> {
+pub fn release_config(endowed_accounts: Vec<MltKeysInfo>) -> Result<ChainSpec, String> {
+    let wasm_binary = WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?;
+    let bootnodes = get_bootnodes();
+
+    // only Alice has sudo powers
+    let sudo = endowed_accounts.first().cloned().ok_or("endowed accounts is empty")?;
+
+    // alice won't be a validator.
+    let mut validator_accounts = endowed_accounts.clone();
+    validator_accounts.remove(0);
+
+    Ok(ChainSpec::from_genesis(
+        // Name
+        "Release",
+        // ID
+        "rel",
+        ChainType::Development, //TODO: should it be custom? I'm not sure about this part.
+        move || {
+            testnet_genesis(
+                wasm_binary,
+                // Initial PoA authorities;
+                validator_accounts.clone(),
+                // Sudo account
+                sudo.controller_account_id(),
+                // Pre-funded accounts;
+                endowed_accounts.clone(),
+                // Pre-fund all accounts in the pallet-balance
+                endowed_accounts.clone(),
+            )
+        },
+        // Bootnodes
+        bootnodes,
+        // Telemetry
+        None,
+        // Protocol ID
+        None,
```

Comment:
WIll have to add protocol ID at some point to avoid collisions with other Substrate-based networks